test/unit/v1.0.0/normalize-v1.test.js

```diff
@@ -1343,6 +1337,37 @@ describe('v1.0.0 normalization', function () {
             });
 
             it('should correctly handle non-string file entities in requests', function (done) {
+                transformer.normalizeSingle({
```

Comment:
can you add a row without value to test if the defaults are added with noDefaults: false

Reply:
@kamalaknn The test after this checks for that.